onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt

```diff
@@ -7,7 +7,7 @@ import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @Tag(name = "Public")
-@RequestMapping("/public")
+@RequestMapping({"/api/public", "/public"})
```

Comment:
@garronej I'm sorry, I've broken the build: https://github.com/InseeFrLab/onyxia-api/runs/4365099636?check_suite_focus=true#step:5:1574

I don't know anything about kotlin ...

Reply:
Me neither lol, I'm looking into it

Reply:
![image](https://user-images.githubusercontent.com/6702424/144023055-537574d4-2ed6-4fa5-a559-7a8eae9a394c.png)

Reply:
it's [here](https://github.com/InseeFrLab/onyxia-api/blob/714c65ca74180980b7093f123e9f094dd5f40587/onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt#L10)